src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -84,6 +84,9 @@ def __setup_cluster_and_execute_test_config(self, config: str) -> int:
             self.additional_cluster_config = self.test_config.integ_test.get("additional-cluster-configs")
             logging.info(f"Additional config found: {self.additional_cluster_config}")
 
+        if self.additional_cluster_config is None:
```

Comment:
how about just initialize `self.additional_cluster_config` as empty directory and check the `len` or `keys()` method to check if it has any values? @peterzhuamazon

Reply:
That can be done but right now I am just matching the OS side implementation.
We can improve this in later PRs.

Reply:
I'm late to this game, but assigning of a default should happen in the constructor, not inside the execution.

Otherwise do `self.additional_cluster_config or {}` anywhere it's used.